src/WebUI/WorkLoads/WorkloadDetails.tsx

```diff
@@ -170,10 +170,53 @@ export class WorkloadDetails extends BaseComponent<IWorkloadDetailsProperties, I
     private _getWorkloadDetails(): JSX.Element | null {
         const metadata = this.props.parentMetaData;
         if (metadata) {
-            const pipeline = Utils.getPipelineText(metadata.annotations);
+            const pipelineDetails = Utils.getPipelineDetails(metadata.annotations);
             const tableItems: IWorkLoadDetailsItem[] = [{ podTemplate: this.props.podTemplate, parentMetaData: metadata, selector: this.props.selector }];
             const agoTime = Date_Utils.ago(new Date(metadata.creationTimestamp), Date_Utils.AgoFormat.Compact);
 
+            const getSubtext = (pipelineDetails: IMetadataAnnotationPipeline): React.ReactNode => {
+                const first = pipelineDetails.jobName
+                    ? localeFormat(Resources.ServiceCreatedWithPipelineText, agoTime, pipelineDetails.jobName)
+                    : localeFormat(Resources.CreatedAgo, agoTime);
+
+                const runElement = pipelineDetails.runName
```

Comment:
should we open in a new tab or in the same window? also we have to make sure the text is blank and on hover or focus only it turn to blue

Reply:
I'll take this in the next PR